### PR TITLE
🧹 Update README

### PR DESCRIPTION
**Pull request by misterpantz**
Also fixed error in OPC UA text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Like all other commands, you can specify different providers like `k8s`, `aws`,
 
 ![](docs/images/cnquery-scan.gif)
 
-These files can also contain multiple query packs for many different target systems. For an example, see `examples/multi-target.mql.yaml`.
```
Comment (tas50) [inline README.md:124]: Is this file gone now?

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ iex ((New-Object System.Net.WebClient).DownloadString('https://install.mondoo.co
 Install-Mondoo;
 ```
 
-If you prefer manual installation, you find the cnquery packages on  [GitHub releases](https://github.com/mondoohq/cnquery/releases).
+If you prefer manual installation, you can find the cnquery packages in our [releases](https://github.com/mondoohq/cnquery/releases).
```
Comment (tas50) [inline README.md]: Any reason not to call it out as being our github releases page?

File: README.md [hunk 3]
```diff
@@ -131,19 +131,17 @@ Go to [console.mondoo.com](http://console.mondoo.com) to sign up.
 
 To learn about Mondoo Platform, read the [Mondoo Platform docs](https://mondoo.com/docs/platform/home/) or visit [mondoo.com](https://www.mondoo.com).
 
-## Private query packs
+## Distribute queries across your fleet with private query packs
 
-Distributing queries across your fleet
+You can create and share query packs using the Registry in the Mondoo Console. The Registry a secure, private environment in your account where you store both Mondoo query packs and custom query packs. This lets you use the same query packs for all assets.
```
Comment (tas50) [inline README.md]:
```suggestion
You can create and share query packs using the Registry in the Mondoo Console. The Registry is a secure, private environment in your account where you store both Mondoo query packs and custom query packs. This lets you use the same query packs for all assets.
```

File: README.md [hunk 4]
```diff
@@ -182,6 +180,7 @@ cnquery bundle upload mypack.mql.yaml
 | Microsoft Azure subscriptions | `azure`                    | `cnquery shell azure --subscription SUBSCRIPTION_ID`                                                                                                        |
 | Microsoft Azure instances     | `ssh`                      | `cnquery shell ssh user@host`                                                                                                                               |
 | Okta                          | `okta`                     | `cnquery shell okta --token TOKEN --organization ORGANIZATION`                                                                                              |
+| Oracle Cloud Infrastructure (OCI) | `oci`                     | `cnquery shell oci`                                                                                              |
```
Comment (tas50) [inline README.md]: This needs some fmt

Comment (tas50) [reply README.md]: We're also missing a few from this list:

  arista           Scan an Arista endpoint.
  opcua            Scan an OPC UA endpoint.

File: README.md (hunk 1 above)
Comment (misterpantz) [reply README.md:124]: Yes. And right now we don't have a replacement that does what this describes.

File: README.md (hunk 2 above)
Comment (misterpantz) [reply README.md]: Not really. I was changing awkward language and in the process I just left it out for brevity. Is there value in pointing out that it's GH?

File: README.md (hunk 3 above)
Comment (misterpantz) [reply README.md]: Thanks!

File: README.md (hunk 2 above)
Comment (tas50) [reply README.md]: We have a lot of different places to get the files so it's a bit nice to be clear what we're talking about this time

Comment (misterpantz) [reply README.md]: Cool beans.

Comment (misterpantz) [inline README.md]:
```suggestion
If you prefer manual installation, you can find the cnquery packages in our [GitHub releases](https://github.com/mondoohq/cnquery/releases).
```

File: README.md (hunk 1 above)
Comment (tas50) [reply README.md:124]: @chris-rock should we nuke this or make a new example file?

**Review by arlimus — APPROVED** [review]
Thank you for the updates!
Please keep an eye on the changes that are landing right now, I'm about to replace all the `./lr markdown` commands in the Makefile, hope I don't miss anything. It's going to be an exciting week

**Pull request merged**